src/main/java/uk/gov/ons/ctp/response/lib/common/time/DateTimeUtil.java

```diff
@@ -1,25 +1,12 @@
 package uk.gov.ons.ctp.response.lib.common.time;
```

Comment:
Is this not being used by schedulers?

Reply:
I couldn't find anything in the codebase that was using the 2 functions I deleted.  Can you see anywhere that it's used?